index.html

```diff
@@ -85,7 +85,7 @@ <h2 class="prt">Tonic
           </li>
         </ul>
         <div class="btl">
-          <button class="btn"><a href="#" class="btt">See Project</a></button>
+          <button class="btn"><p class="btt">See Project</p></button>
         </div>
```

Comment:
- [ ] Since the see project text is already wrapped with a `button` tag, It is better if you remove the `p` here because the `p` tag is only used for paragraphs not buttons.
  - `hint` you can refactor to this :arrow_forward: `<button class="btn btt">See Project</button>`

Reply:
Thanks. made the change suggested.